### PR TITLE
Petras survey

### PR DESCRIPTION
**Pull request by petrasson**

### CODE REVIEW

File: code/src/App.js [hunk 1]
```diff
@@ -1,9 +1,120 @@
...
+
+  /// //Keep track of which question is next////
+  const handleFormSubmit = (e) => {
+    e.preventDefault();
+    const currentQuestionNumber = currentQuestion + 1;
+    setcurrentQuestion(currentQuestionNumber);
+
+    /// //Save answers////
+    const updateAnswer = (updatedAnswer) => {
+      setAnswers(() => [...answers, ...updatedAnswer]);
```
Comment (JosefineEngdahl) [inline code/src/App.js:67]: Haven't seen "..." om row 67 before, curious of what it means :)

File: code/src/App.js [hunk 2]
```diff
@@ -1,9 +1,120 @@
...
+    ]
+  }
+];
 
 export const App = () => {
+  const [currentQuestion, setcurrentQuestion] = useState(0);
+  const [userInput, setUserInput] = useState('');
+  const [answers, setAnswers] = useState([]);
+
+  /// //Keep track of which question is next////
```
Comment (JosefineEngdahl) [inline code/src/App.js:59]: Nice comments through out the code, makes it easy to follow!

File: code/src/components/Summary.js [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+          <h2 className="summary-text">Ok {answers[3]}!</h2>
+          <p>Let&apos;s see if we can you a some nice music based on this:</p>
+          <p>Energy: {answers[5]}</p>
+          <p>Genre: {answers[7]}</p>
+          <p>Epoke: {answers[9]}</p>
+          <p>Click on the button to play the music!</p>
+        </div>
+        <div className="result">
+          <button className="button-result" type="button">
+            <a href="https://open.spotify.com/track/6EvQkav3bbrEAWRmvs6EOj?si=53475e0f4d934c8d">
```
Comment (JosefineEngdahl) [inline code/src/components/Summary.js:17]: Smart to add a link to Spotify!

File: code/src/components/Types.js [hunk 4]
```diff
@@ -0,0 +1,76 @@
+import React from 'react';
```
Comment (JosefineEngdahl) [inline code/src/components/Types.js]: This whole component looks really well structured, easy to follow :)

**Review by JosefineEngdahl — APPROVED** [review]
Great job! I really like your survey and the theme!

It’s hard to feedback on the code since it’s still so new and we built it in different ways. However, I really like the way you’re structuring your code, instead of one component for each question (as I did) you only have three in total which makes it easy to follow and easy to change if you want to add/remove/change a certain question.

The CSS is not fully responsive but I ran it thtough two validator tools and it looks great! :)

File: code/src/App.js (hunk 1 above)
Comment (petrasson) [reply code/src/App.js:67]: Yes, it's something a friend of mine showed me. :) It's just a short-cut. Here is example from stack overflow:



Screenshot 2022-10-05 at 15 23 29

File: code/src/components/Summary.js (hunk 3 above)
Comment (petrasson) [reply code/src/components/Summary.js:17]: Yeah.. when I have time I will make it more fun and a nicer experience. :)

**Review by JosefineEngdahl — APPROVED** [review]

**Pull request closed without merging**